python/dgl/graphbolt/impl/sampled_subgraph_impl.py

```diff
@@ -145,7 +174,20 @@ def exclude_edges(
         should be a pair of tensors representing the edges to exclude. If
         sampled subgraph is heterogeneous, then `edges` should be a dictionary
         of edge types and the corresponding edges to exclude.
-
+    exclude_mode : Union[str, callable], optional
+        Whether and how to exclude dependencies related to the provided edges
+        in the argument ``edges``. Possible values are
+        - If None, for excluding only provided ``edges``.
+        - If "reverse", for excluding not only the provided ``edges`` but aslo
+          their reverse edges. In the case of a homogeneous graph, reverse
+          edges refer to edges with inverted source and destination node IDs.
+          In heterogeneous graphs, this encompasses reversing both node IDs
```

Comment:
There are some corner cases we need to be crystal clear in doc string:
1) If in heterogeneous graphs, the edges (A:E:A) has the same node type for both src and dst, users still need to specify the reverse_etypes map (A:E:A) - (A:E:A), is that right?
2) if in heterogeneous graphs, for edges (A:E:B), users want to exclude all reverse edge (B:*:A) regardless edge type, users need to input all type in reverse_etypes (A:E:B) - (B:*:A), is that right?

Reply:
Totally correct, add this in the docstring for clearity.